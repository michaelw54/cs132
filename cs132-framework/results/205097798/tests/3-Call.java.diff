1,4c1,2
< ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
< Was expecting:
<     "return" ...
<     
---
> 42
> 99
