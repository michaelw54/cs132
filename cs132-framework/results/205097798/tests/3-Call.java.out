ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
Was expecting:
    "return" ...
    
